src/OpenSearch.OpenSearch.Managed/ConsoleWriters/LineOutParser.cs

```diff
@@ -79,14 +79,14 @@ public static class LineOutParser
 			section = match.Groups["section"].Value.Trim().Replace("org.opensearch.", "");
 			node = match.Groups["node"].Value.Trim();
 			message = match.Groups["message"].Value.Trim();
-			started = TryGetStartedConfirmation(section, message);
+			started = TryGetStartedConfirmation(section, message, node);
 			return true;
 		}
 
-		private static bool TryGetStartedConfirmation(string section, string message)
+		private static bool TryGetStartedConfirmation(string section, string message, string node)
 		{
-			var inNodeSection = section == "o.e.n.Node" || section == "node";
-			return inNodeSection && message == "started";
+			var inNodeSection = section == "c.a.o.s.c.ConfigurationRepository";
```

Comment:
Does it work with `OpenSearch` and `OpenDistro` both?

Reply:
Yes. I tried with both